### PR TITLE
Follow-up to showing all contributions

### PR DESCRIPTION
**Pull request by dbrant**
This does a few things:
* Fixes the edit streak shown in the Edits screen by taking into account edits from the user's home wiki.
* In the Contributions activity, it will now show all contributions from the user's home wiki (which we weren't doing yet??)

### CODE REVIEW

File: app/src/main/java/org/wikipedia/userprofile/ContributionsFragment.kt [hunk 1]
```diff
@@ -195,22 +193,38 @@ class ContributionsFragment : Fragment(), ContributionsHeaderView.Callback {
...
+                val cont = response.continuation["uccontinue"]
+                if (cont.isNullOrEmpty()) {
+                    continuations.remove(WikipediaApp.getInstance().wikiSite)
+                } else {
+                    continuations[WikipediaApp.getInstance().wikiSite] = cont
+                }
+                response.query?.userContributions()?.forEach {
+                    contributions.add(Contribution("", it.revid, it.title, it.title, it.title, EDIT_TYPE_GENERIC, null, it.date(),
+                        WikipediaApp.getInstance().wikiSite, 0, it.sizediff, it.top, 0))
+                }
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/userprofile/ContributionsFragment.kt:210]: Looks like the order of all the contributions is showing incorrectly (not ordered by date).

**Review by cooltey — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/userprofile/ContributionsFragment.kt (hunk 1 above)
Comment (dbrant) [reply app/src/main/java/org/wikipedia/userprofile/ContributionsFragment.kt:210]: @cooltey  I can't reproduce. We sort all the contributions by date in the `createConsolidatedList()` function when building the list to be displayed.  Are you totally sure you saw your contributions out of order? Can you post a screenshot?

Comment (cooltey) [reply app/src/main/java/org/wikipedia/userprofile/ContributionsFragment.kt:210]: Ah! My bad. I guess somehow I was confused by Jun and Jul. The order looks good to me now.

However, I am not sure about the edits streak. In my task screen it shows 5days edit streak, but in the contribution screen it doesn't look like I have a 5 days edit streak.

Comment (dbrant) [reply app/src/main/java/org/wikipedia/userprofile/ContributionsFragment.kt:210]: Well, it looks like the streak logic in `getEditStreak()` is just wrong and needs to be fixed. (i.e. not related to this PR)

Comment (dbrant) [reply app/src/main/java/org/wikipedia/userprofile/ContributionsFragment.kt:210]: @cooltey I made a quick fix that should improve the edit streak logic, although it's still naive and not expected to be perfect.

**Pull request merged**